examples/maps/composite_map_AIA_HMI.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+=========================================
+Composite map
+=========================================
+In this example we make a composite map out
+of images provided by AIA and HMI.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+
+import sunpy.map
+import sunpy.data.sample
+
+#this should be gone when the HMI data is added
+#to the examples folder
+from sunpy.net import Fido, attrs as a
+from sunpy.net import jsoc
+
+##############################################################################
+
+#downloading the "missing" data
+#the AIA data timestamp we need to
+#match is 20110607 063302
+def download(tstart = '2011/06/07 06:32:45', tend = '2011/06/07 06:33:15'):
```

Comment:
Why did you create a function for this?

Reply:
Because there is no example file for this hmi fit (which closest corresponds to the AIA map).
When the hmi map is added to the examples folder this would be removed.

Reply:
Sorry,  I wasn't clear. I know why you download the file. Does it need to be a defined function?

Reply:
No. That was just a personal preference.